scripts/zones/Navukgo_Execution_Chamber/mobs/Immortal_Flan.lua

```diff
@@ -0,0 +1,19 @@
+-----------------------------------
```

Comment:
No need for this script as it has no info at this time

Reply:
Should I just delete it?  :)

Reply:
Git delete yes. Unless you want to add something into the lua for the mob

Reply:
Deleted. Pushing change soon.